File: pset1/mario/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
  
    int height;
    do
    {
        printf("Height: ");
        height = get_int();
    }
    while (height < 0 || height >== 23);
    
    int hashes = 2;
    int spaces = height - 1;
    
    for (int i = 0; i < height; i++)
    {
        for(int j = 0; j < spaces; j++)
        {
            printf(" ");
        }
        spaces--;
        
        for(int k = 0; k < hashes; k++)
        {
            printf("#");
        }
        hashes++;
        
        printf("\n");
        
    }
}

File: pset6/mario.py

import cs50

while True:
    height = cs50.get_int('Height: ');
    if height > 0 and height <= 23:
        break

hashes = 2
spaces = height - 1

for i in range(height):
    for j in range(spaces):
        print(' ', end='')
    spaces -= 1
    for k in range(hashes):
        print('#',end='')
    hashes += 1
    print()